packages/amplify_core/lib/src/config/api/api_config.dart

```diff
@@ -30,4 +31,25 @@ class ApiConfig extends AmplifyPluginConfigMap {
 
   @override
   Map<String, Object?> toJson() => _$ApiConfigToJson(this);
+
+  DataOutputs? toDataOutputs({AWSConfigMap<CognitoAppSyncConfig>? appSync}) {
+    final plugin = awsPlugin?.default$;
```

Comment:
I'm not sure if using `default$` is correct as it uses `authorizationType == APIAuthorizationType.apiKey` to get the default plugin. I think we want to get the plugin with `endpointType == EndpointType.graphQL` instead.